tests/test_install_mender_sh.py

```diff
@@ -78,6 +91,26 @@ def run(self, command, warn=False):
     return c
 
 
+def put_all_packages(container, dest):
+    container.run(f"mkdir -p {dest}")
+    for package in glob.glob(packages_path("mender-client", "amd64") + "/*.deb"):
+        container.put(package, dest)
+
+
+def prepare_local_apt_repo(container):
+    # Copy freshly built packages and configure a local APT repo with dpkg-dev. See:
+    # https://askubuntu.com/questions/458748/is-it-possible-to-add-a-location-folder-on-my-hard-disk-to-sources-list
+    put_all_packages(container, "/packages")
+    container.run("apt install -y dpkg-dev")
+    container.run(
+        "cd /packages && dpkg-scanpackages . /dev/null | gzip -9c > Packages.gz"
```

Comment:
Yes! This is exactly what I was talking about

Reply:
Yes, thank you for sharing this with me some time ago - it came in handy right now.